flent/scripts/ss_iterate.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#Copyright (C) 2016  Matthias Tafelmeier
+
+#ss_iterate.sh is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+
+#lindwurm is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+
+#You should have received a copy of the GNU General Public License
+#along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+count=10
+interval=0.1
+host=localhost
+
+while getopts "c:I:H:t:" opt; do
+    case $opt in
+        c) count=$OPTARG ;;
+        I) interval=$OPTARG ;;
+        H) host=$OPTARG ;;
+        t) target=$OPTARG ;;
+    esac
+done
+
+cmd_generic="ss -t -i -p | grep -A 1 $target"
+command_string=$(cat <<EOF
+#which ss_iterate >/dev/null && exec tc_iterate $buffer -i $interface -c $count -I $interval -C $command; // not rdy yet
+for i in \$(seq $count); do
+	TMP_OUT=\`$cmd_generic\`
+	echo "\$TMP_OUT" | egrep -o '\<cwnd:[0-9]*\>' | sed 's/cwnd://g' | column;
+	echo "\$TMP_OUT" | egrep -o '\<rtt:[0-9]*\>' | sed 's/rtt://g' | column;
+	echo "\$TMP_OUT" | egrep -o '\<pid=[0-9]*\>' | sed 's/pid=//g' | column;
```

Comment:
Ok, the netperf matching does muddy waters a little. From optimality perspective, I'm fully aware this is a nightmare, as already pointed out but only **INTERIM**.

Fine, am shifting format lifting into the parser.